src/main.c

```diff
@@ -256,9 +304,14 @@ poe_cmd_queue(unsigned char *_cmd, int len)
 
 	list_add_tail(&cmd->list, &cmd_pending);
 
-	if (empty)
+	if (!empty)
+		return 0;
+	if (config.i2c_bus < 0)
 		return poe_cmd_send(cmd);
 
+	poe_i2c_cmd_send(cmd, reply, sizeof(reply));
+	poe_reply_consume(reply);
```

Comment:
Are we SURE we cannot use `ustream_fd_init` (ustream / uloop) to consume instead?